firebase.js

```diff
@@ -194,10 +194,15 @@ const config = {
         type: 302,
       },
       {
-        destination: '/geoip/legacy',
+        destination: '/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired',
         source: '/product/geoip/geoip-legacy{,/}',
         type: 302,
       },
+      {
+        destination: '/geoip/release-notes/2022#geoip-legacy-databases-have-been-retired',
+        source: '/geoip/legacy',
```

Comment:
Changing the source to `'/geoip/legacy{,/}'` should allow you to delete all the other redirects that were added in this commit.